fbink_device_id.c

```diff
@@ -346,6 +346,21 @@ static bool
 			strtcpy(deviceQuirks.deviceCodename, "Sangria", sizeof(deviceQuirks.deviceCodename));
 			strtcpy(deviceQuirks.devicePlatform, "Bellatrix4", sizeof(deviceQuirks.devicePlatform));
 			return true;
+		case 0XFA0u:    // KS2
+		case 0XFA1u:
+		case 0XFE5u:
+		case 0XF9Du:
+		case 0XFE4u:
+		case 0XFE3u:
+		case 0X102Eu:
+		case 0X102Du:
+			deviceQuirks.isMTK         = true;
+			deviceQuirks.hasEclipseWfm = true;
+			deviceQuirks.screenDPI     = 300U;
+			strtcpy(deviceQuirks.deviceName, "Scribe2", sizeof(deviceQuirks.deviceName));
```

Comment:
```suggestion
			strtcpy(deviceQuirks.deviceName, "Scribe 2", sizeof(deviceQuirks.deviceName));
```

(To match hw the other numbered devices are named ;)).